README.md

```diff
@@ -8,6 +8,52 @@ SumoLogic Distro of [OpenTelemetry Collector][otc_link] built with
 [otc_link]: https://github.com/open-telemetry/opentelemetry-collector
 [otc_builder_link]: https://github.com/open-telemetry/opentelemetry-collector-builder
 
+## Configuration
+
+In order to send data to [Sumo Logic][sumologic_webpage] one needs to configure
+the [sumologicextension][sumologicextension] with credentials and define it (the extension) in the same service
+as [sumologicexporter][sumologicexporter] is defined so that it's used as auth
+extension.
+
+```yaml
+extensions:
+  sumologic:
+    access_id: <my_access_id>
+    access_key: <my_access_key>
+    collector_name: <my_collector_name>
+
+# Define your receivers...
+receivers:
+  hostmetrics:
+    collection_interval: 30s
+    scrapers:
+      load:
+
+# Define your processors...
+processors:
+
+exporters:
+  sumologic:
+    authkj:
+      authenticator: sumologic
+
+service:
+  extensions: [sumologic]
```

Comment:
Can we have that predefined by default so that noone has to add this section?

Reply:
I guess not, if it is possible it would be a hack. Even if it is possible let's discuss it separately and only then change this README